packages/jspsych/src/modules/extensions.ts

```diff
@@ -0,0 +1,14 @@
+export interface JsPsychExtensionParameters {
+  [key: string]: any;
+}
+
+export interface JsPsychExtensionData {
+  [key: string]: any;
+}
+
+export interface JsPsychExtension {
+  initialize(params?: JsPsychExtensionParameters): Promise<void>;
+  on_start(params?: JsPsychExtensionParameters): void;
+  on_load(params?: JsPsychExtensionParameters): void;
+  on_finish(params?: JsPsychExtensionParameters): JsPsychExtensionData;
+}
```

Comment:
Great! :tada: 
Instead of using comments above these methods in the webgazer extension, it would make sense to add JsDoc comments here so we get comments for all future extensions as well and help extension authors doing their jobs :)